sources/packages/backend/apps/api/src/route-controllers/student/student.controller.service.ts

```diff
@@ -64,25 +71,57 @@ export class StudentControllerService {
     groupName: string,
     auditUserId: number,
   ): Promise<FileCreateAPIOutDTO> {
-    const createdFile = await this.fileService.createFile(
-      {
-        fileName: file.originalname,
-        uniqueFileName: uniqueFileName,
-        groupName: groupName,
-        mimeType: file.mimetype,
-        fileContent: file.buffer,
-      },
-      studentId,
-      auditUserId,
-    );
-
-    return {
-      fileName: createdFile.fileName,
-      uniqueFileName: createdFile.uniqueFileName,
-      url: `student/files/${createdFile.uniqueFileName}`,
-      size: createdFile.fileContent.length,
-      mimetype: createdFile.mimeType,
-    };
+    let uploadStatus: number;
+    try {
+      this.logger.log(`Uploading file ${file.originalname} to S3 storage.`);
+      uploadStatus = await this.objectStorageService.putObject({
```

Comment:
Is the `uploadStatus` really needed?
Is it not the case that the method will either put the object with success or throw an error?
Do we really to inspect the HTTP response?